### ISSUE TITLE
Add Multicall Support

### ISSUE THREAD

**Issue by chand1012** [multisig-labs/ceres#4]
One of the issues I had when working on #2 was that I was getting rate limited by the RPC API. A simple solution would be to take the mass of Web3 requests we're making and turn them into Web3 multicalls. Ideally I would like to automatically do this without specifying in the config we want to run a multicall request. Someone has already made a helper for this that's compatible with ethers, see [ethers-multicall](https://github.com/cavanmflynn/ethers-multicall). 

## TLDR

Find a way to implement [ethers-multicall](https://github.com/cavanmflynn/ethers-multicall) into this project without having to modify the dashboard configs.
